ReactAndroid/src/main/jni/react/jni/OnLoad.cpp

```diff
@@ -21,6 +21,7 @@
 #include "WritableNativeArray.h"
 #include "WritableNativeMap.h"
 
+#include <JReactMarker.h>
```

Comment:
Can you change this line for: 

#include "JReactMarker.h"

Reply:
done